modules/core/src/core/transform.js

```diff
@@ -137,7 +138,7 @@ export default class Transform {
 
     this._setupBuffers({sourceBuffers, feedbackBuffers});
     this._setupSwapBuffers();
-    this._buildModel({id, vs, varyings: varyingsArray, drawMode, elementCount});
+    this._buildModel({id, vs, modules, varyings: varyingsArray, drawMode, elementCount});
```

Comment:
The shadermodule system has more parameters than `modules`: `dependencies`, `inject` etc. 

WOuld it not make more sense to pass through the props from `Transform` to the `Model` constructor rather than having to copy them one by one and keep updating when something changes?


```
_initialize(props) {
  const {...} = props;
  ...
  this.buildModel(Object.assign({}, props, {
    // any overrides
  }))
```